RELEASE_NOTES.md

```diff
@@ -1,13 +1,10 @@
 # Release Notes
-Another day, another release. These are the release notes for the version `v1.31.0`.
+Another day, another release. These are the release notes for the version `v1.32.0`.
 
 ## Features
-* feat(cli): add lacework generate cloud-account oci subcommand (#1359) (Kolbeinn)([33722fdc](https://github.com/lacework/go-sdk/commit/33722fdc163f3f14fe6eeafa8f67c7cb142f0b80))
-## Bug Fixes
-* fix(api): fix alert rules source field (#1370) (Pengyuan Zhao)([c54d98d1](https://github.com/lacework/go-sdk/commit/c54d98d162c1b477d1e373e893e9c6226af45604))
+* feat(RAIN-70441): add CLI command to support migration (#1369) (Manan Bhatia)([8d3ed7f2](https://github.com/lacework/go-sdk/commit/8d3ed7f2023a94958ea458d6dccbf6dc5573ee7b))
```

Comment:
Could you remove the RAIN-70441 it doesn't really have a meaning for anyone other than Lacework